app/src/main/res/values/array.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string-array name="languagentries">
-        <item>Default</item>
```

Comment:
this right here

Reply:
Thanks, are you going to fix this or should I take this up and fix it quickly?

Reply:
changing this won't solve the issue, please do proper research before suggesting any changes.
Thanks.